ui/components/Notification/MesherySnackbarWrapper.js

```diff
@@ -0,0 +1,78 @@
+import react, {useState} from "react";
+import { styled } from "@mui/material/styles";
+import {
+    IconButton,
+    Typography,
+    Paper,
+    Collapse,
+    Card,
+    CardActions,
+    Grid
+  } from "@mui/material";   
+  import { SnackbarContent } from 'notistack';
```

Comment:
let's move away from notistack and just use mui snackbar.

Reply:
@YashKamboj ^^

Reply:
@YashKamboj Please update this too.

Reply:
Done!!